### PR TITLE
python310Packages.intake: unstable-2023-08-24 -> 0.7.0

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

Get back to stable release of `intake`.
Now builds fine.

cc @veprbl 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 255872` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/intake/default.nix [hunk 1]
```diff
@@ -26,16 +25,16 @@
...
+  version = "0.7.0";
   format = "setuptools";
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "81b1567a2030adfb22b856b4f63cefe35de68983";
```
Comment (wegank) [inline pkgs/development/python-modules/intake/default.nix:36]: The `rev` doesn't change?

File: pkgs/development/python-modules/intake/default.nix [hunk 2]
```diff
@@ -26,16 +25,16 @@
...
+  disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "81b1567a2030adfb22b856b4f63cefe35de68983";
-    hash = "sha256-S2PoUN0Bao5VULfHhgbXXowopPLm/njAHO3dIM8ILno=";
+    owner = "intake";
+    repo = "intake";
+    rev = version;
```
Comment (veprbl) [inline pkgs/development/python-modules/intake/default.nix:36]:
```suggestion
    rev = "refs/tags/${version}";
```

Comment (GaetanLepage) [reply pkgs/development/python-modules/intake/default.nix]: Done !

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 255872` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake.dist</li>
  </ul>
</details>

**Review by wegank — APPROVED** [review]

**Review by SomeoneSerge — APPROVED** [review]

**Review by NickCao — APPROVED** [review]

**Pull request merged**